reflectometry/amor_reduction.ipynb

```diff
@@ -0,0 +1,306 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "enhanced-issue",
+   "metadata": {},
+   "source": [
+    "# Amor Reduction\n",
+    "\n",
+    "## How to start:\n",
+    "\n",
+    "Before starting you must:\n",
+    "- Have conda installed\n",
+    "- conda env create -f ess-notebooks-latest.yml python=3.7 . The yaml environment file is part of this repository.\n",
+    "\n",
+    "## What will this notebook show?\n",
+    "\n",
+    "The notebook will show how to use the `ess.amor.AmorData`, `ess.amor.AmorReference` and `ess.amor.Normalisation` classes for the reduction of data collected at the Amor instrument at PSI. \n",
+    "To achieve this, we will reduce the following data files, [sample.nxs](./sample.nxs) and [reference.nxs](./references.nxs), and then normalise the sample data (with respect to the reference data). \n",
+    "\n",
+    "## Reduction\n",
+    "\n",
+    "Before, we begin, we should import then necessary modules. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "virgin-intake",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import dataconfig\n",
+    "import numpy as np\n",
+    "import scipp as sc\n",
+    "import scippneutron as scn\n",
+    "from ess.amor import AmorData, AmorReference, Normalisation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "assured-emergency",
+   "metadata": {},
+   "source": [
+    "We can then define some information about the reduction, for inclusion in the final `.ort` file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "hollywood-absolute",
```

Comment:
What are these strange ids??

Reply:
I have no idea 🤣

Reply:
They are apparently randomly generated phrases (https://github.com/jupyter/notebook/issues/6001).

Reply:
Yes, they are new, and discussion/fixes are ongoing because people are having issues with them in diffs:

- https://github.com/jupyterlab/jupyterlab/issues/9729#issuecomment-803197368
- https://github.com/jupyterlab/jupyterlab/issues/9645
- https://github.com/kynan/nbstripout/issues/146

Which version of Jupyter/Lab are you on?